test/memory_import.wat

```diff
@@ -0,0 +1,13 @@
+(module
+  (import "" "memory" (memory 1 2))
+
+  (func (export "size") (result i32) (memory.size))
+  (func (export "load") (param i32) (result i32)
+    (i32.load8_s (local.get 0))
+  )
+  (func (export "store") (param i32 i32)
+    (i32.store8 (local.get 0) (local.get 1))
+  )
+
+  (data (i32.const 0x1000) "\01\02\03\04")
+)
```

Comment:
Is this file still necessary?

Reply:
it is not in a c-inspired test. but I want to use it in a further test. 
should we make those live in a new directory?